packages/http-client-csharp/emitter/src/type/input-examples.ts

```diff
@@ -25,7 +25,7 @@ interface InputExampleBase {
 export interface InputHttpOperationExample extends InputExampleBase {
   kind: "http";
   parameters: InputParameterExampleValue[];
-  responses: Map<number, OperationResponseExample>;
+  responses: OperationResponseExample[];
```

Comment:
status is not needed anymore?

Reply:
our response already has a property for the `status code`